bin/deployment-create-status

```diff
@@ -15,6 +15,13 @@ DEPLOYMENT_ID=$("${BASEDIR}"/deployment-get-id)
 DEPLOY_STATE=$(echo "${1}"|tr "[:upper:]" "[:lower:]")
 DEPLOY_DESC="${2}"
 
+VALID_STATES=("error" "failure" "inactive" "in_progress" "queued" "pending" "success")
+
+if [[ $(echo "${VALID_STATES[@]}" | grep -o "${1}" | wc -w) -eq 0 ]]; then
```

Comment:
Could we perhaps use `${DEPLOY_STATE}` rather than `${1}`? Seems like we do some upper/lower of that in line 15
`DEPLOY_STATE=$(echo "${1}"|tr "[:upper:]" "[:lower:]")`